src/main/java/com/npsproject/it4045npsproject/controller/NPSController.java

```diff
@@ -5,10 +5,13 @@
 import com.npsproject.it4045npsproject.model.Park;
 import com.npsproject.it4045npsproject.service.ParkSearchService;
 import com.npsproject.it4045npsproject.service.ParkService;
+import org.springframework.stereotype.Controller;
 import org.springframework.web.bind.annotation.*;
 import org.springframework.beans.factory.annotation.Autowired;
 
-@RestController
+import java.util.List;
+
+@Controller
```

Comment:
Changed to a controller so that this controller will be able to use models to return http pages.